src/components/application_manager/test/commands/mobile/set_global_properties_test.cc

```diff
@@ -90,6 +94,8 @@ class SetGlobalPropertiesRequestTest
   }
 
   ~SetGlobalPropertiesRequestTest() {
+    // Required for proper DataAccessor release
+    Mock::VerifyAndClearExpectations(&app_mngr_);
```

Comment:
@istoilovgithub is app_manager is static? I am not sure that it should be VerifyAndClearExpectations for it? 
@LevchenkoS please clarify this issue